contrib/debian/rules

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/make -f
+
+# Find the arch we are building for, as this determines
+# the location of plugins in /usr/lib
+DEB_HOST_MULTIARCH ?= $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
+
+%:
+	# For jessie and beyond
+	dh $@ --with systemd
+	# For wheezy use the following (and also remove
+	# dh-systemd Build-Depends from the control file)
+	#dh $@
+
+override_dh_auto_configure:
+	# Build the configure script if it doesn't already exist
+	-[ ! -x $(CURDIR)/configure ] && $(CURDIR)/autogen.sh
```

Comment:
Does [dh-autoreconf](https://wiki.debian.org/Autoreconf#Using_dh-autoreconf) take care of this? Untested, so perhaps I've missed the point.

```
%:
	dh $@  --with autoreconf --with=systemd
```

Requires build-depend on dh-autoreconf.